### DISCUSSION TITLE
Is props jump broken in Volar v1.7.8?

### DISCUSSION THREAD

**Discussion by zheung** [vuejs/language-tools#3234] [General]
Voloar: 1.7.8 or 1.6.5
Vue: 3.3.4
VSCode: 1.79.0-insider

I've been aware of this problem for a long while now, but wasn't sure if it was caused by something I'd changed or not.
It wasn't until I used `npm init vue@latest` to create a _brand new project_ that I realised it was probably a bug.
Props jumping and component-name jumping are not available for either JS or TS projects

I changed the type definition of `msg` prop and added a comment, but it does not reflect in the tips:
![image](https://github.com/vuejs/language-tools/assets/894004/b91f867a-d3b3-4942-b9fe-972a304d7ff5)
![image](https://github.com/vuejs/language-tools/assets/894004/7bd847a4-e64e-4853-9522-04265b1e61f1)

**Comment by xmatthias** [comment]
I think i've got the same problem - but it think it goes further than this what you're reporting.
To me, it's completely missing the whole template section highlighting, as if it's no longer correctly detecting the template components.

A brand-new project scaffolded with vite:
![image](https://github.com/vuejs/language-tools/assets/5024695/53b480a2-a06d-4e6a-9278-eb00fb89ebb3)

Using vscode 1.78.2 - with the volar extension 1.6.4 (notice the different coloring of the tag, as well as the tooltip showing the component definition:

![image](https://github.com/vuejs/language-tools/assets/5024695/26745560-2903-4842-9c54-3b15785a509e)

Using vscode 1.79.0-insider (2023-06-02)
with the extension either volar v1.6.5 or v1.7.8

![image](https://github.com/vuejs/language-tools/assets/5024695/2862680d-26ab-458b-aabc-ef8152c7de2d)


I've been unable to get this to work AT ALL with any vscode insiders version from this week.
I've also already tested with an empty profile (only installed volar into it) - but it doesn't seem to help.

**Reply by xmatthias** [reply]
Seems like this propagated to stable now with the release of 1.79.0, and is still present with both the volar stable plugin, as well as with the pre-release version of 1.7.11.
